website/api/views.py

```diff
@@ -606,3 +608,12 @@ def post(self, request, *args, **kwargs):
             },
             status=200,
         )
+
+
+class CompanyViewSet(viewsets.ModelViewSet):
+    queryset = Company.objects.all()
+    serializer_class = CompanySerializer
+    permission_classes = [IsAuthenticatedOrReadOnly]
+    filter_backends = (filters.SearchFilter,)
+    search_fields = ("id", "name")
+    http_method_names = ["get", "post", "put"]
```

Comment:
Let's use consistent syntax whenever possible.
```suggestion
    permission_classes = (IsAuthenticatedOrReadOnly,)
    filter_backends = (filters.SearchFilter,)
    search_fields = ("id", "name")
    http_method_names = ("get", "post", "put")
```

Reply:
I used the previously made views like the UserProfileViewSet as reference and it was the same there... dont know if this would work.. should i still make this changes?